From 4f2c6d8f9c7110b9128c308d7bd062483ca7b558 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fabiano=20Fid=C3=AAncio?= <fidencio@redhat.com>
Date: Fri, 8 Jan 2016 12:33:34 +0100
Subject: [PATCH] display: set min value for desktop-{width,height} props as
 MIN_DISPLAY_{WIDTH,HEIGHT}

Otherwise we can have warnings when resizing the virt-viewer window to
the smallest possible size, like:

(virt-viewer:11187): GLib-GObject-WARNING **: value "50" of type `gint'
is invalid or out of range for property `desktop-height' of type `gint'

Related: rhbz#1296878
(cherry picked from commit 907f8c5d3347eef95ecb6eccc7c5752093c7f5cf)
---
 src/virt-viewer-display.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/virt-viewer-display.c b/src/virt-viewer-display.c
index 829ee07..2b1f2b0 100644
--- a/src/virt-viewer-display.c
+++ b/src/virt-viewer-display.c
@@ -114,7 +114,7 @@ virt_viewer_display_class_init(VirtViewerDisplayClass *class)
                                     g_param_spec_int("desktop-width",
                                                      "Width",
                                                      "Desktop width",
-                                                     100,
+                                                     MIN_DISPLAY_WIDTH,
                                                      G_MAXINT32,
                                                      100,
                                                      G_PARAM_READWRITE));
@@ -124,7 +124,7 @@ virt_viewer_display_class_init(VirtViewerDisplayClass *class)
                                     g_param_spec_int("desktop-height",
                                                      "Height",
                                                      "Desktop height",
-                                                     100,
+                                                     MIN_DISPLAY_HEIGHT,
                                                      G_MAXINT32,
                                                      100,
                                                      G_PARAM_READWRITE));
